src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -798,10 +802,12 @@ def setUp(self):
         # Start the servers
         force_agent_start = False
         if self.setup_start_servers:
+            self.log_step('setUp(): Starting servers')
```

Comment:
I'm not sure I agree with these because now EVERY test will have these steps. I don't see these as "test steps". For many tests, it's assumed that a DAOS cluster is running before testing something. I.e. not all tests are _**testing**_ server start.
And this also means if `setup_start_servers` is False, the "test steps" are changed. So running one test method from a file vs running multiple test methods in the same file results in different "test steps"

Reply:
Although it is part of setUp for most tests one could consider it a step in terms of running the test.  Ultimately, though the reason I'm adding it is to get some the elapsed time markers in setUp as this is typically where we see variations when tests timeout.  I do agree we should be careful not to let the various sub-steps of starting servers or clients become test steps as well as that could become too much.